tests/unit/editor/editor-test.js

```diff
@@ -354,7 +354,7 @@ test('editor parses HTML post using parser plugins', (assert) => {
   editor = new Editor({html, parserPlugins: [parserPlugin]});
   assert.ok(!!editor.post, 'editor loads post');
 
-  assert.deepEqual(seenTagNames, ['TEXTAREA', 'IMG']);
+  assert.deepEqual(seenTagNames, ['P', 'TEXTAREA', 'IMG']);
```

Comment:
Is this characterized as a bugfix or as a breaking change?

Reply:
I would characterise it as a bugfix because the old behaviour meant elements that were expected to go through parser plugins were never seen. I suppose it may be a breaking change if a consumer has built parser plugins to work around the buggy behaviour, I haven't been able to think up an example of that yet 🤔

Reply:
We talked about this and considered some examples. It seems like it wouldn't break any code we looked at in our quick review, so I'm going to call this a bugfix 👍